SERVER/Server.js

```diff
@@ -160,17 +161,36 @@ server.get(Constants.DEAUTH_REQUEST, async function(req, res) {
 
 server.get(Constants.QUALITY_OVERLAY_REQUEST, function(req, res) {
     var query = req.query;
+    query.minLatitude   = parseFloat(query.minLatitude);
+    query.minLongitude  = parseFloat(query.minLongitude);
+    query.maxLatitude   = parseFloat(query.maxLatitude);
+    query.maxLongitude  = parseFloat(query.maxLongitude);
 
     logger.info("[Server][qualityOverlay] Overlay requested from ("+query.minLatitude+","+query.minLongitude+") to ("+query.maxLatitude+","+query.maxLongitude+")");
 
+    if(query.minLatitude > query.maxLatitude || query.minLongitude > query.maxLongitude){
+        sendErrorMessage(6, req, res);
+        return;
+    }
+    if(query.minLatitude < -90 || query.maxLatitude > 90 || query.maxLongitude > 180 || query.minLongitude < -180){
+        sendErrorMessage(5, req, res);
```

Comment:
Please use error name as opposed to number, it makes reading and debugging the code easier.